src/index.js

```diff
@@ -1,74 +1,26 @@
 import './style.css';
-
-const root = document.getElementById('parent');
-const listTodo = [
-  {
-    description: 'Wake up at 8:00 AM',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Wash my face and brush my teeth.',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Eat breakfast',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Prepare for Microverse class',
-    completed: false,
-    index: 0,
-  },
-];
-
-const createElement = (type, cls, id) => {
-  const element = document.createElement(type);
-
-  if (cls) {
-    for (let i = 0; i < cls.length; i += 1) {
-      element.classList.add(cls[i]);
+import tasks from './tasks.js';
+import Display from './display.js';
+
+const display = new Display();
+
+/* updateTasks = () => {
+    for (let i = 0; i < tasks.taskCollection.length; i += 1) {
+      const description = document.getElementById('desc');
+      description.addEventListener('keypress', (event) => {
+        if (event.key === 'enter') {
+          const task = new OneTask();
+          task.description = description.value;
+          task.completed = chk.value;
+          task.index = tasks.index;
+          tasks.taskCollection.push(task);
+          this.populatePage();
+        }
+      });
     }
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: